addons/base_setup/models/res_config_settings.py

```diff
@@ -43,6 +43,12 @@ class ResConfigSettings(models.TransientModel):
     company_count = fields.Integer('Number of Companies', compute="_compute_company_count")
     active_user_count = fields.Integer('Number of Active Users', compute="_compute_active_user_count")
     language_count = fields.Integer('Number of Languages', compute="_compute_language_count")
+    translation_auto_update = fields.Boolean(
+        "Automatically Update Translations",
+        default=True,
+        config_parameter="base.translation_auto_update",
```

Comment:
Not needing an ir.config.param for this, you can simply override `get_param`/`set_param` (or compute/inverse) that will change the `active` value of the cron